src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -126,6 +126,11 @@ final class SecureStore: Store {
 		set { kvStore["hasSeenSubmissionExposureTutorial"] = newValue }
 	}
 
+	var hasSeenBackgroundFetchAlert: Bool {
```

Comment:
Is this safe when it comes to updates? can we simply add attributes here or do we require DB migration code when we release this?

Reply:
Hi @SebastianWild  @ChristianKienle @haosap, do you know whether this could be an issue?

Reply:
Hi @johannesrohwer ,
I think it should not be an issue. Just assume that, the users update the app, and the callee will get a false.

Reply:
Thanks for the clarification!! 👍